File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        book = {}
        top = []
        for val in nums:
            book[val] = book.get(val,0) +1
        
        for i in range(k):
            highest_val = max(book, key=book.get)
            top.append(highest_val)
            del book[highest_val]
    
        return top
            

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.go
func findMaxConsecutiveOnes(nums []int) int {
	max := 0
	tempMax := 0
	for _,val := range nums {
		if val == 1 {
			tempMax +=1
		} else {
			tempMax = 0
		}
		if tempMax > max {
			max = tempMax
		}
	}
	return max
}
